### PR TITLE
Issue #793: Project release date vs post date in project views on backdropcms.org

### PR DESCRIPTION
**Pull request by bugfolder**
Fixes https://github.com/backdrop-ops/backdropcms.org/issues/793.
Fixes https://github.com/backdrop-ops/backdropcms.org/issues/738.
Fixes https://github.com/backdrop-ops/backdropcms.org/issues/709.

This PR fixes mismatched release date and release number in two places:

Project search results: see, e.g., https://backdropcms.org/modules/?s=Ubercart
Project page: see, e.g., https://backdropcms.org/project/ubercart

In editing the View `project_release_download_table`, I changed the admin names of the displays to be a bit more suggestive of where they appear (to make it easier for the next person trying to figure out where they are used).

And I also edited the top of the sidebar on the project page a bit:

* Removed "Other releases" heading because it's unnecessary (and slightly misleading, since it includes the recommended release)
* Removed extra ">" from "See all versions" link because that's non-standard (no other links on this page have it).
* Added a colon to "Recommended version" to match "Released".

### CODE REVIEW

**Comment by bugfolder** [comment]
To see the effect, compare these two pages on b.org and tugboat:

Project search results:
 https://backdropcms.org/modules/?s=Ubercart
https://pr798-olhxo25vj87gw3m4kltcz0hojgf8qrku.tugboat.qa/modules/?s=Ubercart

Project page:
https://backdropcms.org/project/ubercart
https://pr798-olhxo25vj87gw3m4kltcz0hojgf8qrku.tugboat.qa/project/ubercart

**Review by klonos — APPROVED** [review]
LGTM 👍🏼

**Comment by BWPanda** [comment]
I just made a change to the Tugboat config to fix an issue with PR previews. In order to get the Tugboat preview working, please rebase the PR.

File: www/themes/borg/css/components-nodes.css [hunk 1]
```diff
@@ -115,6 +115,17 @@ label.github-clone-label {
     margin-left: 0;
   }
 }
+.node-type-project-module .block-field-node-field-download-count .field-item,
```
Comment (jenlampton) [inline www/themes/borg/css/components-nodes.css:118]: All these CSS changes are going to be (accidentally) reverted on the next update of the borg theme if they aren't also repeated in https://github.com/backdrop-contrib/borg. 

If they are specific to this site only, they should be moved into the `backdropcms` theme instead.

File: www/themes/borg/css/node-project.css [hunk 2]
```diff
@@ -212,13 +212,19 @@ article.node-project-release.view-mode-project-search article.view-mode-teaser {
 .node-project .view-project-release-download-table th,
 .node-project .view-project-release-download-table td {
   padding: 5px .8em .0 0;
+}
+.node-project .view-project-release-download-table th {
```
Comment (jenlampton) [inline www/themes/borg/css/node-project.css:216]: All these CSS changes are going to be (accidentally) reverted on the next update of the borg theme if they aren't also repeated in https://github.com/backdrop-contrib/borg. 

If they are specific to this site only, they should be moved into the `backdropcms` theme instead.

File: www/themes/borg/template.php [hunk 3]
```diff
@@ -271,7 +271,7 @@ function borg_preprocess_node(&$variables){
       $footer_links['version']['data'] = backdrop_render($version);
 
       // Add the latest release date.
-      $date = format_date($release->node_created, 'short');
+      $date = format_date($release->node_project_release_supported_versions_created, 'short');
```
Comment (jenlampton) [inline www/themes/borg/template.php:274]: This change is going to be (accidentally) reverted on the next update of the borg theme if they aren't also repeated in https://github.com/backdrop-contrib/borg.

**Pull request merged**